lib/env-options.js

```diff
@@ -15,4 +15,8 @@ if (process.env.DOTENV_CONFIG_DEBUG != null) {
   options.debug = process.env.DOTENV_CONFIG_DEBUG
 }
 
+if (process.env.DONENV_CONFIG_OVERWRITE != null) {
```

Comment:
Does this have a typo here? Shouldn't that be `DOTENV_CONFIG_OVERWRITE`?

Reply:
@dstankovd do you have any news?

Reply:
I'm not sure what do you mean. It should check if there is DONENV_CONFIG_OVERWRITE set as env variable on the host machine and use it's value if present. It's covered in test-env-options.js  and works as expected as far as I can tell.

Reply:
I think it's about the typo error (like a grammar or orthographic mistake) in line 18.

It's typed **DONENV**_CONFIG_OVERWRITE, but shouldn't be **DOTENV**_CONFIG_OVERWRITE, with a T?

Reply:
i don't think it's a typo since it's DONENV everywhere, not just here. but i don't know why a N and not a T. maybe a typo at first and after it's the editor autocompletion ?